setup.cfg

```diff
@@ -137,7 +137,6 @@ all =
     impyla
     openlineage-airflow>=0.12.0
     paramiko
-    protobuf<=3.20.0  # Bigquery provider isn't compatible with it. Details in https://github.com/apache/airflow/commit/25a9ae3b2eec85dfd500b0a921045fc95ab8ffd6
```

Comment:
We would also need to remove this from the Google extras here https://github.com/astronomer/astronomer-providers/pull/1126/files#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52R67

Reply:
Can we test that all Google Cloud example DAGs (Especially BigQuery ones since the comment says that it was done for BigQuery) run fine after this change?

Reply:
Yes, I just removed it. We've already tested all the google example DAGS without encountering errors